S7.Net/PLC.cs

```diff
@@ -85,7 +85,7 @@ public bool IsAvailable
             {
                 try
                 {
-                    Connect();
+                    ConnectAsync().Wait();
```

Comment:
Use `ConnectAsync().GetAwaiter().GetResult()` instead. I don't know all the details, but for one it doesn't wrap any exception in an `AggregateException`, which is already enough to warrant the change.